product_docs/docs/pem/9/monitoring_performance/probes.mdx

```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
     To invoke a script on a Windows system, set the registry entry for `AllowBatchJobSteps` to `true` and restart the PEM agent.
 
 -   Use the **Target Type** list to select the object type for the probe to monitor. **Target Type** is disabled if **Collection method** is WMI.
+    probes are used to collect data from different levels of the system hierarchy. The Target Type of a probe defines the specific level within the system hierarchy where the probe will collect data. This ensures that the probe     
+    gathers relevant metrics from the appropriate components of the system.
+
+    <details><summary>Target Type Configuration Matrix</summary>
```

Comment:
This table below will not render. There's no blank line between the HTML and the Markdown table.

Reply:
Review is in progress :-)